functionary/core/models/function.py

```diff
@@ -99,3 +124,7 @@ def render_name(self) -> str:
     def schema(self) -> dict:
         """Function definition schema"""
         return get_schema(self)
+
+    def is_active(self) -> bool:
+        """Returns true if active and the package isn't disabled"""
+        return self.active and self.package.enabled
```

Comment:
There was a conversation at some point lamenting our inconsistency with the status terminology.  It would be nice if we could pick one and go with it.  I'd probably vote for `active` over `enabled`.  Even if internally the terms end up being different for some reason, the public facing thing should probably be the same, such that this is:

```python
    @property
    def is_active(self) -> bool:
        """Returns true if active and the package isn't disabled"""
        return self.active and self.package.is_active
```